Sources/MapboxCoreNavigation/EHorizon/RoadGraphEdgeMetadata.swift

```diff
@@ -17,39 +17,48 @@ extension RoadGraph.Edge {
     /** Edge metadata */
     public struct Metadata {
 
+        // MARK: Geographical & Physical Characteristics
+        
         /** The bearing in degrees clockwise at the start of the edge. */
         public let heading: CLLocationDegrees
 
         /** The edge’s length in meters. */
         public let length: CLLocationDistance
         
+        /** The edge’s mean elevation, measured in meters. */
+        public let altitude: CLLocationDistance?
+
+        /** The edge’s curvature. */
+        public let curvature: UInt
+
+        // MARK: Classification qualities
+        
+        /** Is the edge a bridge? */
+        public let isBridge: Bool
+        
+        /** The number of parallel traffic lanes along the edge. */
+        public let laneCount: UInt?
+        
         /** The edge’s general road classes. */
         public let roadClasses: RoadClasses
         
         /** The edge’s functional road class, according to the [Mapbox Streets source](https://docs.mapbox.com/vector-tiles/reference/mapbox-streets-v8/#road), version 8. */
         public let mapboxStreetsRoadClass: MapboxStreetsRoadClass
 
+        /** Indicates how many directions the user may travel along the edge. */
+        public let directionality: Directionality
+        
+        // MARK: Legal definitions
+        
         /** The edge’s maximum speed limit. */
         public let speedLimit: Measurement<UnitSpeed>?
 
         /** The user’s expected average speed along the edge, measured in meters per second. */
         public let speed: CLLocationSpeed
```

Comment:
These properties and `drivingSide` (and maybe `laneCount`) should go together in a section called “Road Usage” or “Road Regulations”, while `name` and `countryCode` could go in a section called “Identity”.

Reply:
Thanks for the suggestion! I am not a native speaker but `Road Regulations` sounds better to my ear :)